src/frontend/screens/Settings/components/UseFramelessWindow.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { ToggleSwitch } from 'frontend/components/UI'
+import useSetting from 'frontend/hooks/useSetting'
+
+const UseFramelessWindow = () => {
```

Comment:
one thing I just realized is that we probably want to NOT show this option on the Steamdeck in gaming mode, since the minimize/maximize buttons will be broken

we have a `isSteamDeckGameMode` export in constants.ts in the backend, we would need to get this information to the frontend

Reply:
Hmm okay, I don't have a SteamDeck so no way to test this, but does it make sense to add a whole new IPC function `isSteamDeckGameMode` for this case (plus pulling that flag into the `GlobalState` so it can be accessed from `GeneralSettings` without requiring any further contortions to deal with asynchronously polling a flag)? 

Or would it be enough to use the `isFullscreen` flag for this purpose, which is already in the context, and whose value is determined by either `isSteamDeckGameMode` or `isCLIFullscreen` being true (see `main.ts:658`)? Does it make sense to show this option when CLI fullscreen mode is enabled? And should we also make sure not to enable frameless mode in `main_window.ts` if either of these are true?